File: Java/programmers/src/level2/완주하지못한선수.java
import java.util.*;
class Solution {
    public String solution(String[] participant, String[] completion) {
        String answer = "";
        Arrays.sort(participant);
        Arrays.sort(completion);
        // equals는 내용 비교, ==는 주소값 비교
        for (int i=0; i < completion.length; i++) {
            if (!participant[i].equals(completion[i])){
            // if (participant[i] != completion[i]){
                return participant[i];
            }
        }
        return participant[participant.length-1];
    }
}

File: Programmers/level1/완주하지못한선수.py
# collections.Counter를 사용하면 participant의 요소별 개수를 딕셔너리 형태로 표시
import collections


def solution(participant, completion):
    print(collections.Counter(participant))
    print(collections.Counter(completion))
    answer = collections.Counter(participant) - collections.Counter(completion)
    return list(answer.keys())[0]